CalculateCurrentIssues.cls

```diff
@@ -1,71 +1,71 @@
 Global class CalculateCurrentIssues implements Schedulable{
 
-global void execute(SchedulableContext SC) {
+global void execute(SchedulableContext sc) {
      ClearAllIssueStatuses();
      SetNewIssueStatuses();
   }
 
 public static void ClearAllIssueStatuses() {
- list<issue__c> all_current_issues = [select id from issue__c where issue_status__c != ''];
+ list<issue__c> allCurrentIssues = [select id from issue__c where issue_status__c != ''];
 
- for (issue__c this_issue : all_current_issues) {
-     this_issue.issue_Status__c  = '';
+ for (issue__c thisIssue : allCurrentIssues) {
+     thisIssue.issue_Status__c  = '';
      }
- update all_current_issues;
+ update allCurrentIssues;
 }
 
 public static void SetNewIssueStatuses() {
 
-Set<string> pub_names = new Set<string>();
+Set<string> pubNames = new Set<string>();
 
-list<AggregateResult> find_publications = [select publication_name__c from issue__c where deadline__c >= :date.today() group by publication_name__c];
+list<AggregateResult> findPublications = [select publication_name__c from issue__c where deadline__c >= :date.today() group by publication_name__c];
 
-for (AggregateResult pub_list : find_publications) {
- pub_names.add((string)pub_list.get('publication_name__c'));
+for (AggregateResult pubList : findPublications) {
+ pubNames.add((string)pubList.get('publication_name__c'));
 }
 
-system.debug(pub_names);
+system.debug(pubNames);
 
-for (string this_pub : pub_names ) {
+for (string thisPub : pubNames ) {
 
- list<issue__c> all_pub_new_issues = [select id
+ list<issue__c> allPubNewIssues = [select id
```

Comment:
This SOQL statement is inside a loop and is highly likely to hit governor limits. Please move the query outside of the loop and put records in collections for processing.